content/md/user-management/configure-saml-sso.md

```diff
@@ -0,0 +1,121 @@
+For the `Service Provider`, the Entity ID is automatically generated and corresponds by default to the metadata URL of the SP.
+
+### Assertion Consumer Service (ACS)
+
+The ACS is the SP endpoint (URL) that is responsible for receiving the SAML response from the IdP.
+
+### Metadata
+
+Metadata is a set of information supplied either by the IdP or the SP, in XML format.
+* `Service Provider`: The SP supplied metadata provide the Entity ID, the X.509 certificate used for decrypting messages coming from the SP, the Logout URL and the ACS URL.
+
+:::tips
+The SP metadata information can be accessed at the following URL: https://YOUR_PIM_SERVER/saml/metadata. Depending on your IdP, this metadata URL can be imported into the IdP to ease the configuration process of your application.
+:::
+
+* `Identity Provider`: The IdP can also supply metadata to help configure the SAML communication. At this time, it is not possible to import it in the PIM to configure the Single Sign-On.
+
+## What is SAML?
+
+SAML (Security Assertion Markup Language) is an XML-based standard for exchanging authentication data between an `Identity Provider (IdP)` (such as Microsoft Azure AD) and a `Service Provider (SP)` (in our case our beloved PIM).
+
+# Gather information from your Identity Provider (IdP)
+
+First of all, before configuring your PIM to communicate with an `Identity Provider`, you will have to gather some information about your IdP. Your IT department/SSO administrator must probably have all the information.
+
+# Configure your Service Provider (SP) in your PIM to communicate with the IdP
+
+## Activate/Disable the Single Sign-On feature
+
+### Activate the Single Sign-On feature
+
+To activate the Single Sign-On feature, just click on the `SSO enabled` button in the interface.
+Once enabled, all the authentication requests will be redirected to the authentication server you specified.
+
+### Disable the Single Sign-On feature
+
+To disable the Single Sign-On feature, just click on the `SSO enabled` button in the interface.
+Once disabled, all the authentication requests will go through the classical login interface.
+
+## Configure the Identity Provider (IdP) section
+
+Using the information you gather previously, you can fill all the required fields that are required to communicate with the IdP.
+The mandatory values are the following:
+* `Entity ID`: String that uniquely identify your IdP (it is in general provided by your IdP)
+* `Sign-on URL`: URL that will be used when a user tries to log in the PIM
+* `Logout URL`: URL that will be used when a user ask for logout action in the PIM
+* `Public certificate`: As the communications are encrypted between the SP and the IdP, the IdP public key is required in order to decrypt any incoming message.
+
+:::info
+Certificates must be in X.509 format.
+:::
+
+## Configure the Service Provider (SP) section
+
+The information required for configuring the SP are the following:
```

Comment:
Would it be a bit clearer if you mention here that that information is populated by the PIM and should be used in the IdP (if I understood well).